packages/shared/src/components/post/PostActions.tsx

```diff
@@ -52,7 +52,13 @@ export function PostActions({
       ? ({ id, vote }) => {
           const updatePost = updateCachedPagePost(feedQueryKey, queryClient);
 
-          return mutateVoteFeedPost({ id, vote, items, updatePost });
+          return mutateVoteFeedPost({
+            id,
+            vote,
+            items,
+            updatePost,
+            entity: UserVoteEntity.Post,
```

Comment:
Do we need to send the entity here?
Does the votePost hook not handle it already

Reply:
Its not part of the hook, it is the mutation handler. I adjusted coming type as alternative since it was not used really.

Reply:
Ah ok, hard to follow from the half visible PR interface :)